### PR TITLE
airshare: add page

### PR DESCRIPTION
**Pull request by Moritz921**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 0.1.6**

### CODE REVIEW

File: pages/common/airshare.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+`airshare {{CODE}} {{path/to/file}}`
+
+- Receive a file from airshare:
+
+`airshare {{CODE}}`
+
+- Start airshare with the option to upload:
+
+`airshare --upload {{CODE}}`
+
```
Comment (acuteenvy) [inline pages/common/airshare.md:17]:
```suggestion

- Send files or directories to a receiving server:

`airshare --upload {{code}} {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`

```

File: pages/common/airshare.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- Send files whose paths have been copied to the clipoard:
+
+`airshare --file-path {{CODE}}`
```
Comment (acuteenvy) [inline pages/common/airshare.md:20]:
```suggestion
- Send files whose paths have been copied to the clipboard:

`airshare --file-path {{code}}`
```

File: pages/common/airshare.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Receive a file and copy it to the clipboard:
+
+`airshare --clip-receive {{CODE}}`
```
Comment (acuteenvy) [inline pages/common/airshare.md:24]:
```suggestion
`airshare --clip-receive {{code}}`
```

File: pages/common/airshare.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# airshare
+
+> Transfer data between two machines in a local network.
+> More information: <https://airshare.readthedocs.io>.
```
Comment (acuteenvy) [inline pages/common/airshare.md:4]: I think it's better to point directly to the docs of the cli tool.
```suggestion
> More information: <https://airshare.rtfd.io/en/latest/cli.html>.
```

**Review by acuteenvy — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**